tests/gcp/deployment/google_folder/vars.tf

```diff
@@ -0,0 +1,5 @@
+variable sub_folder_parent {
```

Comment:
How easy will it be to change the variable?

Reply:
Easy, either change the value in the workflow or add a tfvars file. This only references a parent created within the `folders` module though https://github.com/mesoform/Multi-Cloud-Platform-Foundations/blob/d0ffb4142e8a942ca60229932dc3a1b8877bc0f9/tests/gcp/deployment/google_folder/main.tf#L9

Reply:
I'm only asking because I not sure we want to be able to do this on the tests. They should fairly strictly be in a restricted testing folder.

Reply:
It would be within the testing folder. The `folders` module would create some folders within the testing module: https://github.com/mesoform/Multi-Cloud-Platform-Foundations/blob/3f5dc2bcaacafe1755c0eab55e25a43fdafc840d/tests/gcp/deployment/google_folder/main.tf#L1-L4

And the `sub_folders` module would be a child folder of one of the folders in the `folders` module. The variable, just allows selection of which of the `folders` folders the sub_folder is to be a child of, by default it will be the one called `staging`.
https://github.com/mesoform/Multi-Cloud-Platform-Foundations/blob/3f5dc2bcaacafe1755c0eab55e25a43fdafc840d/tests/gcp/deployment/google_folder/main.tf#L6-L10

Since `var.sub_folder_parent` is the name of the folder, it would only be able to reference a folder created by a terraform module. The variable is also used in `folder_import_script.sh` to get the id of the `sub_folder` parent from the `displayName` , but the `gcloud` command requires a `--folder` flag which defines the folder to search for the specified folder. So the `TF_VAR_sub_folder_parent` variable would still need to point to a folder within the  `GCP_FOLDER_ID` folder (i.e. the testing folder): https://github.com/mesoform/Multi-Cloud-Platform-Foundations/blob/3f5dc2bcaacafe1755c0eab55e25a43fdafc840d/tests/gcp/deployment/google_folder/folder_import_script.sh#L39
So if the variable was `"staging"`, that command would get the id of the folder within the testing folder that has the display name staging. That id would then be used as the  `PARENT_ID` for the `--folder` argument when getting the `sub_folder` id for the the import https://github.com/mesoform/Multi-Cloud-Platform-Foundations/blob/3f5dc2bcaacafe1755c0eab55e25a43fdafc840d/tests/gcp/deployment/google_folder/folder_import_script.sh#L27